packages/react/src/useAction.ts

```diff
@@ -77,6 +77,28 @@ export const useAction = <
     F["variablesType"]
   >(plan.query);
 
+  return [
+    processResult(result, action),
+    useCallback(
```

Comment:
Good call on `useCallback`. We don't know how people will use this, so giving this function a stable identity is the better default.